products/green_fast_track/models/product/_product_models.yml

```diff
@@ -189,10 +189,10 @@ models:
       - name: geom
         data_type: geometry(Geometry, 2263)
 
-  - name: green_fast_track_pilot_projects
+  - name: test__actual__pilot_projects
```

Comment:
super nit: should just be one underscore after actual, right?

Reply:
I was looking to mimic our conventions with other model prefixes like `int_flags__`, so `test_actual__` would probably be more aligned than doubles before and after

and paraphrasing from dbt docs on "why double underscores at all": double underscores help visually distinguish separate parts of multi-word names

Reply:
`test_actual__` and `test_expected__` seem good. (Wouldn't nitpick this usually, but when I go copy-pasta, just want to make sure I don't grab `test__expected_pilot_projects`)